modules/libkernel/types.h

```diff
@@ -91,3 +91,9 @@ struct rusage_t {
   uint32_t         ru_nvcsw    = 0;
   uint32_t         ru_nivcsw   = 0;
 };
+
+struct timesec {
```

Comment:
Name it SceTimesce pls. So, it's not getting mixed up with any later internal timesec when porting.